pipeline/sources/lamassu.py

```diff
@@ -273,6 +299,15 @@ def _enforce_columns(df: pd.DataFrame, column_names: dict[str, ExtensionDtype])
         for column, dtype in column_names.items():
             if column not in df:
                 df[column] = pd.Series(dtype=dtype) if dtype else None
+            elif df[column].dtype != dtype:
+                try:
+                    # Coerce to intended dtype. This is needed as e.g. for
+                    # columns with NA values, the column type
+                    # otherwise would be created with float64.
+                    df[column] = df[column].astype(dtype)
+                except Exception:
+                    logging.error(f'Error enforcing type {dtype} for column {column}')
```

Comment:
Just making sure: Does this error-log contain enough context to allow debugging later, e.g. the feed name/URL?